homeassistant/components/workday/binary_sensor.py

```diff
@@ -138,8 +138,10 @@ def setup_platform(
         _LOGGER.debug("No holidays to remove or invalid holidays")
 
     _LOGGER.debug("Found the following holidays for your configuration:")
-    for remove_holiday, name in sorted(obj_holidays.items()):
-        _LOGGER.debug("%s %s", remove_holiday, name)
+    for holiday_date, name in sorted(obj_holidays.items()):
```

Comment:
This was never listing removed holidays as implied previously but was listing all remaining holidays